lib/svgo.test.js

```diff
@@ -64,6 +64,127 @@ test('allow to disable and customize plugins in preset', () => {
   `);
 });
 
+describe('allow to configure EOL', () => {
+  test('should respect EOL set to LF', () => {
+    const svg = `
+      <?xml version="1.0" encoding="utf-8"?>
+      <svg viewBox="0 0 120 120">
+        <desc>
+          Not standard description
+        </desc>
+        <circle fill="#ff0000" cx="60" cy="60" r="50"/>
+      </svg>
+    `;
+    // using toEqual because line endings matter in these tests
+    expect(
+      optimize(svg, {
```

Comment:
Could you please split result into separate variable. This is how I refactored all cases in other branch
```
const { data } = optimize(...)
```

Reply:
Addressed in latest push.